### ISSUE TITLE
Running into post errors with the release endpoint

### ISSUE THREAD

**Issue by jonhilgart22** [anton-yurchenko/git-release#16]
### Description:  
A clear and concise description of what the bug is
![image](https://user-images.githubusercontent.com/8621935/78925367-29792900-7a50-11ea-967c-41912698a531.png)

### Tag:
`ml-fast-api-server0.1.0`

### Workflow:
```name: release

on:
  push:
    tags: #anytime a new tag is pushed that matches this pattern, trigger the workflow
     - "ml-fastapi-server/*"

jobs:
  build:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v1
      - name: Get the version from the github tag
        id: get_version
        run: echo ::set-env name=RELEASE_VERSION::${GITHUB_REF##*/}
      - name: Echo the version tag
        run: |
          echo ${{ env.RELEASE_VERSION }}
      - name: Release
        uses: docker://antonyurchenko/git-release:latest
        env:
            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
            DRAFT_RELEASE: "false"
            PRE_RELEASE: "false"
            CHANGELOG_FILE: "ml-fastapi-server/CHANGELOG.md"
            ALLOW_EMPTY_CHANGELOG: "false"
            ALLOW_TAG_PREFIX: "true"
        with:
          args: |
            ml-fastapi-server/dist/ml_fastapi_server-${{ env.RELEASE_VERSION }}-py3-none-any.whl
            ml-fastapi-server/dist/ml-fastapi-server-${{ env.RELEASE_VERSION }}.tar.gz
```

### Changelog:
```bash
## [0.1.0] - 2020-04-07

## Added

- First PR to test changelog

### Changed

- Nothing

### Removed

- Nothing
```

**Comment by jonhilgart22** [comment]
I'm not sure if this is an issue with this action or with my permissions on github

**Comment by jonhilgart22** [comment]
@anton-yurchenko any ideas?

**Comment by anton-yurchenko** [comment]
Hello @jonhilgart22, 
There was indeed a bug with tags containing `/` that was fixed in today's release [v3.1.2](https://github.com/anton-yurchenko/git-release/releases/tag/v3.1.2).

Please test again with **v3.1.2**/**latest**.

Thanks for raising this issue. 👍
